library/core/src/primitive_docs.rs

```diff
@@ -308,9 +308,10 @@ mod prim_never {}
 /// let _ = unsafe { char::from_u32_unchecked(0x110000) };
 /// ```
 ///
-/// USVs are also the exact set of values that may be encoded in UTF-8. Because
-/// `char` values are USVs and `str` values are valid UTF-8, it is safe to store
-/// any `char` in a `str` or read any character from a `str` as a `char`.
+/// USVs are also the exact set of values that may be encoded in UTF-8. Because `char` values are
```

Comment:
I realise this is pre-existing text, so feel free to ignore, but how about just spelling out "Unicode scalar value" here instead of a novel initialization form that's only used twice?

Reply:
It's used 4 times. :shrug: